### DISCUSSION TITLE
Auto close NeogitLogView panel

### DISCUSSION THREAD

**Discussion by ycycyyc** [NeogitOrg/neogit#1610] [Q&A] [answered]
I'd like to find a way to Automatically close the NeogitLogView panel after some operations, such as 'Reset hard'.

**Comment by CKolkey** [comment, marked as answer]
You can make an autocmd for the "NeogitReset" event that closes the window for the log view's filetype, though this behaviour might be useful enough to just include it standard

**Reply by ycycyyc** [reply]
thanks
